README.md

```diff
@@ -59,14 +59,25 @@ You can run the demo with Marblerun in standalone mode as follows:
     pf_crypt encrypt --input plain/saved_model.pb --output models/resnet50-v15-fp32/1/saved_model.pb --wrap-key model_key
     ```
 
-1. Set the model key in Marblerun's manifest
+1. Generate a user certificate and key.
     ```bash
-    cat tf-server-manifest.json | sed "s|YOUR_KEY_HERE|$(hexdump -ve '1/1 "%02x"' model_key)|g" > manifest.json
+    openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes -keyout user_credentials.key -out user_credentials.crt
     ```
 
-1. Upload the manifest
+1. Insert the output of the following command as `Certificate` for user `tf-admin` in `tf-server-manifest.json`
     ```bash
-    marblerun manifest set manifest.json $MARBLERUN
+    awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' user_credentials.crt
+    ```
+
+1. Upload the manifest:
+    ```bash
+    marblerun manifest set tf-server-manifest.json $MARBLERUN
+    ```
+
+1. Upload the model key to Marblerun.
+    ```bash
+    sed -i "s|KEY_DATA|$(cat model_key | base64)|g" pf_key.json
```

Comment:
We already have the direct PEM upload, no? If so, please use this here for simplicity:-)

Reply:
This is true, but comes with a couple problems:
1. The current implementation expects the PEM file to be a certificate / private key for a certificate.
We could maybe change this, to also allow symmetric-key and plain, but would need to define our own PEM types for this (e.g `SYMMETRIC KEY` and `PLAIN`) or reuse existing types.
2. The model key is saved as a 16 byte binary key, we would need to convert it to PEM pretty much the same way we convert it to json format, not really saving time or making things more clear.

Reply:
Ah I see. Could make sense to have a general --from-file option instead of the --from-pem and detect pem files automatically or alternatively have both options. But that's another ticket.